modules/document_repository/ajax/documentEditUpload.php

```diff
@@ -54,12 +54,12 @@
     if ($action == 'upload') {
         $puser      = $_POST['user'];
         $category   = $_POST['category'];
-        $site       = $_POST['site'];
-        $instrument = $_POST['instrument'];
-        $pscid      = $_POST['pscid'];
-        $visit      = $_POST['visit'];
-        $comments   = $_POST['comments'];
-        $version    = $_POST['version'];
+        $site       = $_POST['site']       !== '' ? $_POST['site'] : null;
+        $instrument = $_POST['instrument'] !== '' ? $_POST['instrument'] : null;
+        $pscid      = $_POST['pscid']      !== '' ? $_POST['pscid'] : null;
+        $visit      = $_POST['visit']      !== '' ? $_POST['visit'] : null;
+        $comments   = $_POST['comments']   !== '' ? $_POST['commnets'] : null;
+        $version    = $_POST['version']    !== '' ? $_POST['version'] : null;
```

Comment:
@driusan @kongtiaowang I usually prefer to ONLY submit to the insert function the non-null values instead of submitting values to be null in SQL. but this works fine on loris from what I can tell. so it passes